content/docs/user-guide/components/reference/gameplay/look-at.md

```diff
@@ -0,0 +1,36 @@
+---
+linkTitle: Look-At
+title: Look-At Component
+description: Use the Look-At component to force an entity to face another entity in your Open 3D Engine (O3DE) level.
+---
+
+Use the **Look-At** component to force an entity to face another entity or position in your Open 3D Engine (O3DE) level.
+
+## Provider
+
+[O3DE Core (LmbrCentral) Gem](/docs/user-guide/gems/reference/o3de-core)
+
+## Look-At properties
+
+![Look-At component properties](/images/user-guide/components/reference/gameplay/look-at-component.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Target** | Sets a target entity, the **Forward Axis** of the entity with the Look-At component will always point towards the **Target** entity. | EntityId | None |
+| **Forward Axis** | Sets the axis that will always point towards **Target** entity. | `Y+`, `Y-`, `X+`, `X-`, `Z+`, `Z-` | None |
+
+## LookAt
```

Comment:
This one is an oddity, I'm not sure how it appears in SC, but in the source the bus has the truncated name 'LookAt' and that is how it is reflected in behavior context/called in Lua.

Reply:
Yeah, they're labelled the same in SC as in Lua. There's inconsistency throughout the product in the translation from bus class to bus reflection name, unfortunately. I suppose there's a good argument for using the reflected name. I wonder if it would be valuable to reformat our template to include _both_ names, at some point.

Reply:
There is the **Scriptable** heading, I opted to only update scriptable methods in this pass (I think this came up in the PR for the shape components).  I have seen an example (in the shape component too?) of a C++ only method with **Scriptable** denoted as 'No'.